packages/base/Avatar/src/Avatar/AvatarWrapper/AvatarWrapper.tsx

```diff
@@ -1,42 +1,80 @@
 import type { ReactNode } from 'react'
 import React from 'react'
 import type { BaseProps, SizeType } from '@toptal/picasso-shared'
-import { twMerge } from '@toptal/picasso-tailwind-merge'
+import { twJoin, twMerge } from '@toptal/picasso-tailwind-merge'
+import { Logo } from '@toptal/picasso-logo'
 
-import { classBySizeAndVariant, clipClassBySize } from './styles'
+import {
+  classBySize,
+  clipClassBySize,
+  logoClassByPadding,
+  logoClassBySize,
+} from './styles'
 
 export type Size = SizeType<'xxsmall' | 'xsmall' | 'small' | 'medium' | 'large'>
 
-export type Variant = 'square' | 'portrait' | 'landscape'
-
 export interface Props extends BaseProps {
   children: ReactNode
   size: Size
-  variant: Variant
+  emblem?: boolean
 }
 
+const showLogo = (size: Size, emblem?: boolean): boolean => {
+  const isTooSmall = ['xsmall', 'xxsmall'].includes(size)
+
+  if (isTooSmall || !emblem) {
+    return false
+  }
+
+  return true
+}
+
+const AvatarLogo = ({ size }: Pick<Props, 'size'>) => (
+  <div
+    className={twJoin('flex absolute bottom-0', logoClassByPadding[size])}
+    role='img'
+    aria-label='photo placeholder'
+  >
+    <Logo
+      emblem
+      variant='blue'
+      // Necessary to decrease the default minWeight and minHeight so that the new width and height could apply
+      // eslint-disable-next-line no-inline-styles/no-inline-styles
+      style={{ minWidth: '1px', minHeight: '1px' }}
```

Comment:
To avoid inline styles, you can also use `base` prop for icons and set it to `1`, but unfortunately this prop is not passed down to icons in `@toptal/picasso-logo`. It could be added, but I'm honestly not sure if it's worth adding a new prop to a component v.s. having inline styles :smile: 
```suggestion
      base={1}
```

Reply:
Yes, that was my first approach as well, but then I thought we want to have as few changes as possible, so I decided to go with the inline solution. If this is accepted then I prefer to go as it is.